knora-ontologies/knora-base.ttl

```diff
@@ -1473,7 +1459,7 @@
 
 ###  http://www.knora.org/ontology/knora-base#valueHas
 
-:valueHas rdf:type owl:DatatypeProperty ;
+:valueHas rdf:type rdf:Property ;
```

Comment:
For example, `valueHasListNode` has to be an object property, because its object is a`ListNode`, not a literal. It can't be a subproperty of `hasLinkTo`, because both the subject and object of `hasLinkTo` must be instances of `Resource`.